requirements.txt

```diff
@@ -1,6 +1,6 @@
 Cython>=0.25
-h5py>=2.6
-numpy>=1.15
+h5py>=2.6,<2.11.0
+numpy>=1.15,<1.19.0
```

Comment:
Should this be platform-specific?

```
numpy>=1.15 ; sys_platform == "Windows"
numpy>=1.15,<1.19.0 ; sys_platform != "Windows"
```

Reply:
Oh well, didn't even know it was possible!
That should be done for neuropod too, since it's not available on windows and fails installation of ludwig[full]

Reply:
Yes, I agree, that's a good idea.